### PR TITLE
 created some external links in README.md

### PR DESCRIPTION
**Pull request by NilmaPeiris**
As the readme.md is too long I created some external links to increase the navigation. Think it will help users to find the particular section they really need without wasting time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ The plugin manager downloads plugins and their dependencies into a folder so tha
 
 ### Usage
 
+- [Getting Started](####Getting Started)
```
[the remark this answers is not in the document]
Comment (NilmaPeiris) [reply README.md:12]: Oh I'll correct it thank you

[the remark this answers is not in the document]
Comment (NilmaPeiris) [reply README.md]: that Example link is working, right?

[the remark this answers is not in the document]
Comment (NilmaPeiris) [reply README.md]: Actually, one # is also not working :-(

[the remark this answers is not in the document]
Comment (NilmaPeiris) [reply README.md]: @timja Now it's working, right??

**Review by timja — APPROVED** [review]

**Pull request merged**